### ISSUE TITLE
Error: EINVAL: invalid argument, uv_pipe_open

### ISSUE THREAD

**Issue by maxduke** [NyaMisty/alt-anisette-server#2]
I've tried this for several hours. Below is the log which I think is most closed to success.
-----------------------------------------------

If after 60 seconds you are still not seeing "Anisette Server running at http://0.0.0.0:6969/"
Please re-run the docker container as the initialize process are stucked

-----------------------------------------------
+ set -e
+ cd /app
+ sleep 10
+ wine 'C:\Program Files\Common Files\Apple\Internet Services\iCloud.exe'
dbus[54]: arguments to dbus_connection_send_with_reply_and_block() were incorrect, assertion "connection != NULL" failed in file ../../../dbus/dbus-connection.c line 3544.
This is normally a bug in some application using the D-Bus library.

  D-Bus not built with -rdynamic so unable to print a backtrace
wine: Assertion failed at address F7F07559 (thread 0150), starting debugger...
+ timeout 45s wine 'Z:\ahk\AutoHotkeyU32.exe' 'Z:\app\icloud_login.ahk'
[i] Initializing
[i] Finding iCloud Window
[i] Finding Account Input
[i] Sending Account Name
[i] Finding Password Input
[i] Sending Password
[i] Clicking login
[i] Waiting for error dialog
[i] got msg: There was an error connecting to the Apple ID server.
[i] Got first-time connection error, wait for locked msg!
[i] Clicking login
[i] Waiting for error dialog
[i] got msg: This Apple ID has been�locked�for security reasons. Visit iForgot to reset your account (https://iforgot.apple.com).
[i] Got locked ret msg, good!
[i] iCloud Initialized successfully!
+ sleep 1
+ export NODE_SKIP_PLATFORM_CHECK=1
+ NODE_SKIP_PLATFORM_CHECK=1
+ wine 'Z:\app\server.exe'
net.js:330
      err = this._handle.open(fd);
                         ^

Error: EINVAL: invalid argument, uv_pipe_open
    at new Socket (net.js:330:26)
    at createWritableStdioStream (internal/bootstrap/switches/is_main_thread.js:72:18)
    at process.getStdout [as stdout] (internal/bootstrap/switches/is_main_thread.js:122:12)
    at console.get (internal/console/constructor.js:204:42)
    at console.value (internal/console/constructor.js:322:50)
    at console.log (internal/console/constructor.js:358:61)
    at Server.<anonymous> (Z:\app\server.js:56:11)
    at Object.onceWrapper (events.js:421:28)
    at Server.emit (events.js:315:20)
    at emitListeningNT (net.js:1352:10) {
  errno: -4071,
  code: 'EINVAL',
  syscall: 'uv_pipe_open'
}
-----------------------------------------------

**Comment by NyaMisty** [comment]
Sorry for late reply, try new version 
(use `docker pull` to update the image before retrying)

**Comment by maxduke** [comment]
Thank you! I can confirm now this is working in Debian 11 with kernel 5.10.0. It's an interesting implementation for Linux.

Just for your information, before the latest fix, I've tried below:
Windows VPS => works fine with those exes
Debian 9 => failed before AHK running
Debian 10 => failed before AHK running
Debian 11 => After AHK running
Then, I realized that wine must depend on the Linux kernel version.
